win32ss/user/user32/windows/cursoricon.c

```diff
@@ -1148,6 +1149,16 @@ BITMAP_LoadImageW(
         goto end;
     CopyMemory(pbmiCopy, pbmi, iBMISize);
 
+    /* Test if MSVC Resource Compiler used and if so, adjust pvBits */
+    HdrSize = *(char*)pbmi;
```

Comment:
Why is there a char cast here when the size is 4 bytes? Can't you just get it from the biSize member?

Reply:
We can just get the value from the biSize member as you suggest and it simplifies the code by allowing us to remove the HdrSize variable. I have modified the code accordingly. Thanks Lots.